### ISSUE TITLE
Testsuit no longer has the tests count attribute

### ISSUE THREAD

**Issue by roniger** [palmerj3/jest-junit#32]
It seems that in the latest release (3.2.1) the format of the xml report file has changed, omitting the tests attribute of the testsuit containing the amount of tests that were ran.
This breaks our CI since our Jenkins publisher is relying on this attribute.
Looking at the code, it seems that there's no way to configure the reporter to include it.
Is there a plan to introduce it back again?

**Comment by palmerj3** [comment]
@roniger could you provide me with a simple example XML that shows how it used to be and how it currently is?

I just want to make sure I understand the issue properly.

Thanks!

**Comment by roniger** [comment]
Old (notice the tests attribute on the testsuite node):
```
<testsuites name="jest tests">
  <testsuite name="Testsuite" tests="4" errors="0" failures="0" skipped="0" timestamp="2017-11-15T20:34:04" time="1.128">
    <testcase classname="Test class" name="Test 1" time="0.03">
    </testcase>
    <testcase classname="Test class" name="Test 2" time="0.023">
    </testcase>
    <testcase classname="Test class" name="Test 3" time="0.069">
    </testcase>
    <testcase classname="Test class" name="Test 4" time="0.054">
    </testcase>
  </testsuite>
</testsuites>
```

New: 
```
<testsuites name="jest tests">
  <testsuite name="Testsuite" errors="0" failures="0" skipped="0" timestamp="2017-11-15T20:34:04" time="1.128">
    <testcase classname="Test class" name="Test 1" time="0.03">
    </testcase>
    <testcase classname="Test class" name="Test 2" time="0.023">
    </testcase>
    <testcase classname="Test class" name="Test 3" time="0.069">
    </testcase>
    <testcase classname="Test class" name="Test 4" time="0.054">
    </testcase>
  </testsuite>
</testsuites>
```

**Comment by palmerj3** [comment]
@roniger great thanks! Let me take a look and see if I can add this back.

**Comment by roniger** [comment]
Thank you!
That would be great

**Comment by palmerj3** [comment]
@roniger just published 3.3.0 which fixes this. Also here is the PR https://github.com/palmerj3/jest-junit/pull/33

Thanks for reporting!!

**Comment by roniger** [comment]
Tested and works great!
Thank you very much for the prompt fix
